charts/cluster-registry-client/values.yaml

```diff
@@ -53,3 +35,36 @@ readinessProbe:
   initialDelaySeconds: 5
   periodSeconds: 10
 terminationGracePeriodSeconds: 10
+
+clusterRegistryClient:
+  alertmanagerWebhook:
+    bindAddress: 0.0.0.0:9092
+    alertMap: []
+  health:
+    healthProbeBindAddress: :9091
+  metrics:
+    bindAddress: 0.0.0.0:9090
+  webhook:
+    port: 9443
+  leaderElection:
+    leaderElect: true
+    resourceNamespace: cluster-registry
+    resourceName: 0c4967d2.registry.ethos.adobe.com
+
+rbac:
+  create: true
+
+serviceAccount:
+  # Specifies whether a service account should be created, else use: 'name: "default"''
+  create: true
+  # The name of the service account to use.
+  # If set, override, else will use chart fullname template
+  name: cluster-registry-client
+
+podDisruptionBudget:
+  enabled: true
```

Comment:
would there be cases where we would not want this to be enabled?

Reply:
Might be, e.g. if someone wants to BYO his own PDB